Software/src/LedDeviceAdalight.hpp

```diff
@@ -64,4 +65,5 @@ public slots:
 	QByteArray m_writeBuffer;
 	QString m_portName;
 	int m_baudRate;
+	QTimer* m_lastWillTimer{nullptr};
```

Comment:
any particular reason to make this a pointer instead of a member? Currently it's not being cleaned up.

Reply:
Indeed, I forgot to delete.
I started with a normal member instance, but it refused to work because of threads, until I used `new` and passed `this`